openmc/model/model.py

```diff
@@ -151,44 +206,156 @@ def from_xml(cls, geometry='geometry.xml', materials='materials.xml',
         settings = openmc.Settings.from_xml(settings)
         return cls(geometry, materials, settings)
 
-    def deplete(self, timesteps, chain_file=None, method='cecm',
-                fission_q=None, **kwargs):
+    def init_lib(self, threads=None, geometry_debug=False, restart_file=None,
+                 tracks=False, output=True, event_based=None, intracomm=None):
+        """Initializes the model in memory via the C API
+
+        .. versionadded:: 0.13.0
+
+        Parameters
+        ----------
+        threads : int, optional
+            Number of OpenMP threads. If OpenMC is compiled with OpenMP
+            threading enabled, the default is implementation-dependent but is
+            usually equal to the number of hardware threads available
+            (or a value set by the :envvar:`OMP_NUM_THREADS` environment
+            variable).
+        geometry_debug : bool, optional
+            Turn on geometry debugging during simulation. Defaults to False.
+        restart_file : str, optional
+            Path to restart file to use
+        tracks : bool, optional
+            Write tracks for all particles. Defaults to False.
+        output : bool
+            Capture OpenMC output from standard out
+        event_based : None or bool, optional
+            Turns on event-based parallelism if True. If None, the value in
+            the Settings will be used.
+        intracomm : DummyCommnicator, mpi4py.MPI.Intracomm or None, optional
```

Comment:
DummyCommunicator is really an implementation detail that shouldn't be exposed to a user. Passing `None` is the right thing to do if you're not using MPI
```suggestion
        intracomm : mpi4py.MPI.Intracomm or None, optional
```